docs/getting-started.rst

```diff
@@ -323,6 +323,8 @@ Docker Compose Install
         ``git clone https://github.com/DefectDojo/django-DefectDojo.git``
 #. Change directories into the newly created folder.
         ``cd django-DefectDojo``
+#. Switch to `dev` branch
+        ``git checkout out dev``
```

Comment:
Typo in git command; should be `git checkout dev`.

Reply:
Actually, we don't need to switch to dev anymore, that was just temporary. So if you want to modify and specify master.